proposals/0000-rich-errors-proposal.rst

```diff
@@ -255,6 +304,33 @@ be useful. We hope that the discussion that arises from this proposal will shed
 light on additional items. Moreover, we anticipate that the vocabulary will
 grow in time as new tooling applications are found.
 
+A smaller but very concrete challenge is figuring out how to give users a hook
```

Comment:
We should be clear about who the "user" is here: API users (right?)

Reply:
Yes. As well as GHC devs, technically, but since we consume this from within GHC we can make arrangements in the code that external (GHC API) users can't.